code/modules/jobs/job_types/wasteland.dm

```diff
@@ -1410,26 +1411,29 @@ Raider
 	name = "Crusader"
 	backpack_contents = list(
 		/obj/item/clothing/suit/armor/light/knight = 1,
-		/obj/item/clothing/head/helmet/f13/knighthelmet = 1,
-		///obj/item/melee/onehanded/machete = 1,
+		/obj/item/clothing/head/helmet/f13/knighthelmet = 1
 	)
 
 /datum/outfit/loadout/samaritan
 	name = "Samaritan"
 	backpack_contents = list(
 		/obj/item/reagent_containers/food/snacks/store/bread/plain = 5,
 		/obj/item/reagent_containers/food/snacks/fishmeat/salmon = 2,
-		/obj/item/reagent_containers/glass/beaker/waterbottle = 2,
-		/obj/item/reagent_containers/glass/beaker/waterbottle/wataur = 1
+		/obj/item/reagent_containers/glass/beaker/waterbottle = 3
 	)	//Matthew 14:17 RSVCE and KJV - Kitsunemitsu
 				//Nerd shit, wataur time. ~TK
+				//Wataur is forbidden. - Yuffels
 
 /datum/outfit/loadout/cleanser
 	name = "Cleanser"
 	backpack_contents = list(
-		/obj/item/grenade/homemade/coffeepotbomb = 2,	//This is funny. I swear guys.
-		///obj/item/gun/ballistic/revolver/m29 = 1,		//Moved here for more *variety*
-		///obj/item/ammo_box/m44 = 2
+		/obj/item/grenade/homemade/coffeepotbomb = 2	//This is funny. I swear guys.
+	)
+
+	/datum/outfit/loadout/newcanaan
```

Comment:
You need to slide this back a bit, it's a nested absolute path when you press TAB on this line like this.

Reply:
Ye, I am blind.